### ISSUE TITLE
AttributeError: 'KMeans' object has no attribute 'show'

### ISSUE THREAD

**Issue by Gdkmak** [DistrictDataLabs/yellowbrick#1088]
**Describe the bug**
I am getting this AttributeError: 'KMeans' object has no attribute 'show' when implementing Elbow Method for KMeans clustering. The elbow graph is plotted after showing the error. 

**To Reproduce** 
```python
from sklearn.cluster import KMeans
from sklearn.datasets import make_blobs
from yellowbrick.cluster import KElbowVisualizer

# Generate synthetic dataset with 8 random clusters
X, y = make_blobs(n_samples=1000, n_features=12, centers=8, random_state=42)

# Instantiate the clustering model and visualizer
model = KMeans()
visualizer = KElbowVisualizer(model, k=(4,12))

visualizer.fit(X)        # Fit the data to the visualizer
visualizer.show()        # Finalize and render the figure
``` 
**Traceback**
```
<ipython-input-69-f375a0067df6> in <module>()
     14 
     15 visualizer.fit(X)        # Fit the data to the visualizer
---> 16 visualizer.show()     # Finalize and render the figure

/usr/local/lib/python3.6/dist-packages/yellowbrick/utils/wrapper.py in __getattr__(self, attr)
     40     def __getattr__(self, attr):
     41         # proxy to the wrapped object
---> 42         return getattr(self._wrapped, attr)

AttributeError: 'KMeans' object has no attribute 'show'
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Python Version 3.6, google colab
 - Yellowbrick Version 0.9.1

**Additional context**
Nope.

**Comment by rebeccabilbro** [comment]
Hi @Gdkmak and thanks for checking out Yellowbrick! 

The tl;dr  version
run `pip install -U yellowbrick` to upgrade your version to the latest.

Detailed explanation: 
It looks like you're on an old version of Yellowbrick. The [`show` method](https://www.scikit-yb.org/en/latest/changelog.html#hotfix-1-0-1) was not introduced until v1.0.1.  Also, I believe version 0.9.1 of YB had a bug originating from scikit-learn's [original misspelling](https://github.com/scikit-learn/scikit-learn/pull/12211/files/b07354da07de036d91f4ec43a5e7d05d91af6614) of the [Calinski Harabasz score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.calinski_harabasz_score.html), which, when resolved by sklearn, triggered the error `ImportError: cannot import name 'calinski_harabaz_score'` after trying to import KElbowVisualizer. Upgrading to the latest Yellowbrick should resolve your issue!

Thanks again for checking out our library! Going to go ahead and mark this resolved; please feel free to reopen if the instructions don't resolve the problem 😄
